### ISSUE TITLE
Arg: cannot correctly align docstring for Tuple option

### ISSUE THREAD

**Issue by vicuna** [ocaml/ocaml#7515]
**Original bug ID:** 7515
**Reporter:** @nojb
**Status:** closed (set by @alainfrisch on 2017-06-30T08:31:24Z)
**Resolution:** fixed
**Priority:** normal
**Severity:** minor
**Fixed in version:** 4.06.0 +dev/beta1/beta2/rc1
**Category:** standard library

## Bug description

The Arg.align function allows to correctly align docstrings of the form

  -foo FOO      My docstring for foo
  -bar BAR      My docstring for bar

by using the docstring "FOO My docstring for foo" in the first case and "BAR My docstring for bar" in the second case.

Unfortunately this does not allow to document Tuple arguments which have more than argument component, e.g.

  -tuple FOO BAR    My docstring for foo

In other words, it would be nice to be able to somehow include spaces in the "first word" of the docstring.

I am not sure what a good solution would be that would not break backwards compatibilty.

Maybe we could declare a not-very-common character to be rendered as a space when found in the "first word" of the docstring.

**Comment by vicuna** [comment]
**Comment author:** @nojb

Mantis stripped the spaces from my description; several spaces are supposed to be present before "My docstring..." in each case.

**Comment by vicuna** [comment]
**Comment author:** @nojb

Possible solution at https://github.com/ocaml/ocaml/pull/1147

**Comment by vicuna** [comment]
**Comment author:** @nojb

This can be closed, as the PR has been merged.

**Comment by vicuna** [comment]
**Comment author:** @damiendoligez

Fixed in trunk by #1147. Thanks @nojebar !
